ts/repositories.bzl

```diff
@@ -6,60 +6,20 @@ See https://docs.bazel.build/versions/main/skylark/deploying.html#dependencies
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
-load("//ts/private:versions.bzl", TS_VERSIONS = "VERSIONS")
+load("//ts/private:npm_typescript_repository.bzl", "npm_typescript_repository", _http_archive_version = "http_archive_version")
+load("//ts/private:versions.bzl", _LATEST_VERSION = "LATEST_VERSION", _TS_VERSIONS = "VERSIONS")
+
+TS_VERSIONS = _TS_VERSIONS
+LATEST_VERSION = _LATEST_VERSION
```

Comment:
Should probably call this `LATEST_TS_VERSION`. Will be a breaking change.

Reply:
We call it `LATEST_VERSION` in all the rules - I think I changed this the other way to be consistent. Users can (and I have done) `load("@rules_ts", typescript_version = "LATEST_VERSION")`